packages/babel-parser/CHANGELOG.md

```diff
@@ -57,7 +57,7 @@ See the [Babel Changelog](https://github.com/babel/babel/blob/master/CHANGELOG.m
 ## 7.0.0-beta.7 (2017-03-22)
 
 ### Spec Compliancy
-* Remove babylon plugin for template revision since it's stage-4 (#426) (Henry Zhu)
+* Remove babel-parser plugin for template revision since it's stage-4 (#426) (Henry Zhu)
```

Comment:
I'm sorry, I think we need to clarify there this terminology is important. The only places we should be using `babel-parser` instead of `@babel/parser` are when we're talking about the directory paths in the repository, I'd expect the vast majority of the time, references to `babylon` would be talking about the package, not about the monorepo folder, so the majority of references should be `@babel/parser`.

Reply:
yes you are right i missed it i guess